pom.xml

```diff
@@ -265,6 +266,12 @@
                 <scope>import</scope>
             </dependency>
 
+            <dependency>
```

Comment:
Can you make sure that transitive dependencies to Infinispan from Vert.x are excluded?